server/repository/src/db_diesel/assets/asset.rs

```diff
@@ -207,42 +183,6 @@ fn create_filtered_query(filter: Option<AssetFilter>) -> BoxedAssetQuery {
         apply_equal_filter!(query, catalogue_item_id, asset_dsl::asset_catalogue_item_id);
         apply_date_filter!(query, installation_date, asset_dsl::installation_date);
         apply_date_filter!(query, replacement_date, asset_dsl::replacement_date);
-
-        if let Some(category_id) = category_id {
```

Comment:
We still want these filters on Asset, the point of using a sub query was to be able to look up the information from the `asset_catalogue_item` table. Was it not working for you? The tests were passing before...